packages/workspace/src/utilities/ts-config.ts

```diff
@@ -0,0 +1,37 @@
+import { offsetFromRoot, Tree, workspaceRoot } from '@nrwl/devkit';
```

Comment:
Let's move these utils over to `@nrwl/js`.

Reply:
done. move them to packages/js/src/utils/typescript/ts-config.ts